### PR TITLE
Add conference timer

### PR DESCRIPTION
**Pull request by theunafraid**
This PR adds a timer that starts after the first participant joined the meeting. The timer represents the duration of the meeting formatted using dateUtil function. Timer should show under the meeting subject.

Web screenshot :
Screenshot 2020-01-10 at 11 58 22

Mobile screenshot :
Screenshot_20200113-110942

### CODE REVIEW

**Comment by saghul** [comment]
Please add a description in the commit message about what this accomplishes.

**Comment by damencho** [comment]
Jenkins test this please.

**Comment by theunafraid** [comment]
Added safeguard and updated mobile app.

File: resources/prosody-plugins/mod_conference_duration_component.lua [hunk 1]
```diff
@@ -0,0 +1,66 @@
+local st = require "util.stanza";
+local socket = require "socket";
+local json = require "util.json";
+local ext_events = module:require "ext_events";
+local it = require "util.iterators";
+
+-- we use async to detect Prosody 0.10 and earlier
+local have_async = pcall(require, "util.async");
+if not have_async then
+    module:log("warn", "speaker stats will not work with Prosody version 0.10 or less.");
```
Comment (damencho) [inline resources/prosody-plugins/mod_conference_duration_component.lua]: Maybe just fix the message to not say 'speaker stats' :)

**Comment by damencho** [comment]
@theunafraid Can you rebase, please, as we just merged a PR that modifies prosody.cfg.lua-jvb.example, sorry about that.
Can upload one web and one mobile screenshot please if it is not too much of a hassle to do that :) Thanks.

File: resources/prosody-plugins/mod_conference_duration_component.lua (hunk 1 above)
Comment (theunafraid) [reply resources/prosody-plugins/mod_conference_duration_component.lua]: Fixed.

**Comment by marclaporte** [comment]
Small but nice new feature. Thank you!

Reminds me of:
https://github.com/sboots/meetingcostcalculator
which would funny to add :-)

**Pull request merged**